koin/src/main/java/in/koreatech/koin/ui/userinfo/UserInfoEditActivity.kt

```diff
@@ -0,0 +1,165 @@
+    private val userInfoEditViewModel by viewModels<UserInfoEditViewModel>()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(binding.root)
+
+        initView()
+        initViewModel()
+
+        userInfoEditViewModel.getUserInfo()
+    }
+
+    private fun initView() = with(binding) {
+        koinBaseAppBar.setAppBarButtonClickedListener(
+            leftButtonClicked = {
+                onBackPressed()
+            },
+            rightButtonClicked = {
+                userInfoEditViewModel.updateUserInfo(
+                    name = binding.userinfoeditedEdittextName.textString,
+                    nickname = binding.userinfoeditedEdittextNickName.textString,
+                    separatedPhoneNumber = arrayOf(
+                        binding.userinfoeditedEdittextPhoneNum1,
+                        binding.userinfoeditedEdittextPhoneNum2,
+                        binding.userinfoeditedEdittextPhoneNum3
+                    ).map { it.textString },
+                    gender = when {
+                        binding.userinfoeditedRadiobuttonGenderMan.isChecked -> Gender.Man
+                        binding.userinfoeditedRadiobuttonGenderWoman.isChecked -> Gender.Woman
+                        else -> null
+                    },
+                    studentId = binding.userinfoeditedEdittextStudentId.textString
+                )
+            }
+        )
+
+        userinfoeditedEdittextStudentId.addTextChangedListener {
+            userInfoEditViewModel.getDept(it.toString())
+        }
+
+        userinfoeditedButtonNicknameCheck.setOnClickListener {
+            userInfoEditViewModel.checkNickname(userinfoeditedEdittextNickName.textString)
+        }
+    }
+
+    private fun initViewModel() {
+        with(userInfoEditViewModel) {
+            withLoading(this@UserInfoEditActivity, this)
+
+            observeLiveData(user) { user ->
+                if (user != null) {
+                    with(binding) {
+                        userinfoeditedTextviewId.text =
+                            getString(R.string.koreatech_email_postfix, user.portalAccount)
+                        userinfoeditedTextviewAnonymousNickName.text = user.anonymousNickname
+
+                        userinfoeditedEdittextName.apply {
+                            setText(user.name)
+
+                            if (user.name == null || user.name!!.isEmpty()) {
+                                isEnabled = true
+                                setDefaultBackground()
+                            } else {
+                                isEnabled = false
+                                setTransparentBackground()
+                            }
+                        }
+
+                        userinfoeditedEdittextNickName.setText(user.nickname)
+
+                        if (user.phoneNumber == null || user.phoneNumber!!.isEmpty()) {
```

Comment:
```suggestion
                        if (user.phoneNumber?.isEmpty() == false) {
                        
                        } else {
                        
                        }
```

로 바꾸는게 좋을것 같습니다. `!!` 는 가급적 사용하지 마세요